source/documentation/tools/quicksight/01-index.md

```diff
@@ -0,0 +1,35 @@
+# QuickSight
+
+⚠️ This service is in beta ⚠️
+
+QuickSight is a data visualisation service that allows you to create interactive dashboards and reports from data available to the Analytical Platform.
+
+We’re still in beta so we'd love to get some of you using QuickSight to get your feedback to help guide data visualization best practices in the Ministry of Justice and make sure we can continue to improve the user experience.
```

Comment:
Is there a feedback mechanism (or placeholder if it's to be decided) to put here so it's clear how people can provide it?

Reply:
James has just added it, so yep! Linked.